README.md

```diff
@@ -4,6 +4,14 @@
 
 [Get it on the Visual Studio Code Marketplace!](https://marketplace.visualstudio.com/items?itemName=ms-vscode-deploy-azure.azure-deploy)
 
+
+## 📢 Attention - Deprecation notice

```

Comment:
![image](https://user-images.githubusercontent.com/4964038/157824189-4a399a53-65fc-4efe-85cb-4ceebba3fefa.png)
Is there a way to highlight it in red to catch the eye? The content looks good to me.

Reply:
Could not find a quick way to add color. I have increased the font, changed to CAPS and added one more emoji ⛔